lib/cast.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-FileCopyrightText: 2022-2024, Alejandro Colomar <alx@kernel.org>
```

Comment:
Since this is a new file I guess this should include only 2024, and not 2022

Reply:
It's new for this project, but I copied from some other projects of mine that were older.  Since I've tweaked it a little bit for this project, I updated the copyright, but without removing the original one.